lib/index.js

```diff
@@ -10,23 +10,41 @@ const Joi = require('joi');
 const Schema = require('./schema');
 const Package = require('../package.json');
 
-// const SchwiftyMigration = require('schwifty-migration');
-
 const internals = {};
 
 exports.register = function (server, options, next) {
 
+    Joi.assert(options, Schema.plugin, 'Bad plugin options passed to schwifty.');
+
     const rootState = internals.state(server.root);
 
-    internals.state(server).dir = options.dir;
     const knexFile = options.knexFile;
```

Comment:
It's the file used for native knex migrations: http://knexjs.org/#knexfile so it's still useful apart from Schwifty, like if you were going to run migration commands with knex via the knex cli